### PR TITLE
Feature/DTGB-475: Timeline issues fix.

### PR DESCRIPTION
**Pull request by meirege**
## Screenshots (if appropriate):
screenshot 2018-08-01 09 12 19

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/components/41-organisms/timeline/timeline.twig [hunk 1]
```diff
@@ -184,13 +186,16 @@
         </div>
       </div>
       <div class="timeline-slot-content accordion--content" id="timeline-slot-content--7">
-        <div class="video-wrapper">
+      <div class="video-wrapper">
```
Comment (Jeroen005) [inline styleguide/components/41-organisms/timeline/timeline.twig]: indent zit nog niet goed hier

File: styleguide/components/41-organisms/timeline/timeline.twig [hunk 2]
```diff
@@ -205,14 +210,16 @@
...
+        <div class="video-inner-wrapper">
           {% include '@video' with {
             'title': 'Promofilm stad Gent.',
             'src': 'https://www.youtube.com/embed/S0vLmTWUmok',
             'audio_description_src': 'https://www.youtube.com/embed/S0vLmTWUmok'
           } %}
         </div>
       </div>
+      </div>
```
Comment (Jeroen005) [inline styleguide/components/41-organisms/timeline/timeline.twig]: Indent

**Review by Jeroen005 — CHANGES_REQUESTED** [review]

File: styleguide/components/41-organisms/timeline/timeline.twig (hunk 1 above)
Comment (meirege) [reply styleguide/components/41-organisms/timeline/timeline.twig]: Fixed.

File: styleguide/components/41-organisms/timeline/timeline.twig (hunk 2 above)
Comment (meirege) [reply styleguide/components/41-organisms/timeline/timeline.twig]: Fixed.

**Review by Jeroen005 — APPROVED** [review]

**Pull request merged**